src/aiida_quantumespresso/calculations/pw.py

```diff
@@ -111,6 +111,8 @@ def define(cls, spec):
                     'scheduler before the files were safely written to disk for a potential restart.')
         spec.exit_code(350, 'ERROR_UNEXPECTED_PARSER_EXCEPTION',
             message='The parser raised an unexpected exception: {exception}')
+        spec.exit_code(360, 'ERROR_G_PAR',
+            message='The code failed in finding a valid reciprocal lattice vector.')
```

Comment:
Can you please explain when/why this error occurs and why this is in 3xx instead of 46x like the others?